<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>916</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>1000</width>
    <height>880</height>
   </size>
  </property>
  <property name="baseSize">
   <size>
    <width>1000</width>
    <height>880</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>1000</width>
     <height>880</height>
    </size>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="0" alignment="Qt::AlignHCenter|Qt::AlignVCenter">
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="minimumSize">
       <size>
        <width>800</width>
        <height>750</height>
       </size>
      </property>
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="start">
       <widget class="QPushButton" name="stayButton">
        <property name="geometry">
         <rect>
          <x>160</x>
          <y>150</y>
          <width>230</width>
          <height>340</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>36</pointsize>
         </font>
        </property>
        <property name="focusPolicy">
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="text">
         <string>매장 이용</string>
        </property>
       </widget>
       <widget class="QPushButton" name="takeOutButton">
        <property name="geometry">
         <rect>
          <x>410</x>
          <y>150</y>
          <width>230</width>
          <height>340</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>36</pointsize>
         </font>
        </property>
        <property name="focusPolicy">
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="toolTipDuration">
         <number>0</number>
        </property>
        <property name="text">
         <string>메뉴 포장</string>
        </property>
       </widget>
       <widget class="QPushButton" name="audioButton">
        <property name="geometry">
         <rect>
          <x>160</x>
          <y>510</y>
          <width>480</width>
          <height>60</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>24</pointsize>
         </font>
        </property>
        <property name="focusPolicy">
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="text">
         <string>음성으로 안내</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="main">
       <widget class="QStackedWidget" name="menuStackedWidget">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>80</y>
          <width>760</width>
          <height>450</height>
         </rect>
        </property>
        <property name="currentIndex">
         <number>5</number>
        </property>
        <widget class="QWidget" name="menu1">
         <widget class="QTableWidget" name="tableWidget_1">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>760</width>
            <height>450</height>
           </rect>
          </property>
          <property name="rowCount">
           <number>4</number>
          </property>
          <property name="columnCount">
           <number>5</number>
          </property>
          <attribute name="horizontalHeaderVisible">
           <bool>false</bool>
          </attribute>
          <attribute name="horizontalHeaderCascadingSectionResizes">
           <bool>false</bool>
          </attribute>
          <attribute name="horizontalHeaderMinimumSectionSize">
           <number>100</number>
          </attribute>
          <attribute name="horizontalHeaderDefaultSectionSize">
           <number>126</number>
          </attribute>
          <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
           <bool>false</bool>
          </attribute>
          <attribute name="verticalHeaderVisible">
           <bool>false</bool>
          </attribute>
          <attribute name="verticalHeaderCascadingSectionResizes">
           <bool>false</bool>
          </attribute>
          <attribute name="verticalHeaderMinimumSectionSize">
           <number>10</number>
          </attribute>
          <attribute name="verticalHeaderDefaultSectionSize">
           <number>140</number>
          </attribute>
          <row/>
          <row/>
          <row/>
          <row/>
          <column/>
          <column/>
          <column/>
          <column/>
          <column/>
         </widget>
        </widget>
        <widget class="QWidget" name="menu2">
         <widget class="QTableWidget" name="tableWidget_2">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>760</width>
            <height>450</height>
           </rect>
          </property>
          <property name="rowCount">
           <number>4</number>
          </property>
          <property name="columnCount">
           <number>5</number>
          </property>
          <attribute name="horizontalHeaderVisible">
           <bool>false</bool>
          </attribute>
          <attribute name="horizontalHeaderMinimumSectionSize">
           <number>100</number>
          </attribute>
          <attribute name="horizontalHeaderDefaultSectionSize">
           <number>126</number>
          </attribute>
          <attribute name="verticalHeaderVisible">
           <bool>false</bool>
          </attribute>
          <attribute name="verticalHeaderMinimumSectionSize">
           <number>10</number>
          </attribute>
          <attribute name="verticalHeaderDefaultSectionSize">
           <number>140</number>
          </attribute>
          <row/>
          <row/>
          <row/>
          <row/>
          <column/>
          <column/>
          <column/>
          <column/>
          <column/>
         </widget>
        </widget>
        <widget class="QWidget" name="menu3">
         <widget class="QTableWidget" name="tableWidget_3">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>760</width>
            <height>450</height>
           </rect>
          </property>
          <property name="rowCount">
           <number>4</number>
          </property>
          <property name="columnCount">
           <number>5</number>
          </property>
          <attribute name="horizontalHeaderVisible">
           <bool>false</bool>
          </attribute>
          <attribute name="horizontalHeaderMinimumSectionSize">
           <number>100</number>
          </attribute>
          <attribute name="horizontalHeaderDefaultSectionSize">
           <number>126</number>
          </attribute>
          <attribute name="verticalHeaderVisible">
           <bool>false</bool>
          </attribute>
          <attribute name="verticalHeaderMinimumSectionSize">
           <number>1</number>
          </attribute>
          <attribute name="verticalHeaderDefaultSectionSize">
           <number>140</number>
          </attribute>
          <row/>
          <row/>
          <row/>
          <row/>
          <column/>
          <column/>
          <column/>
          <column/>
          <column/>
         </widget>
        </widget>
        <widget class="QWidget" name="menu4">
         <widget class="QTableWidget" name="tableWidget_4">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>760</width>
            <height>450</height>
           </rect>
          </property>
          <property name="rowCount">
           <number>4</number>
          </property>
          <property name="columnCount">
           <number>5</number>
          </property>
          <attribute name="horizontalHeaderVisible">
           <bool>false</bool>
          </attribute>
          <attribute name="horizontalHeaderMinimumSectionSize">
           <number>100</number>
          </attribute>
          <attribute name="horizontalHeaderDefaultSectionSize">
           <number>126</number>
          </attribute>
          <attribute name="verticalHeaderVisible">
           <bool>false</bool>
          </attribute>
          <attribute name="verticalHeaderDefaultSectionSize">
           <number>140</number>
          </attribute>
          <row/>
          <row/>
          <row/>
          <row/>
          <column/>
          <column/>
          <column/>
          <column/>
          <column/>
         </widget>
        </widget>
        <widget class="QWidget" name="menu5">
         <widget class="QTableWidget" name="tableWidget_5">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>760</width>
            <height>450</height>
           </rect>
          </property>
          <property name="rowCount">
           <number>4</number>
          </property>
          <property name="columnCount">
           <number>5</number>
          </property>
          <attribute name="horizontalHeaderVisible">
           <bool>false</bool>
          </attribute>
          <attribute name="horizontalHeaderMinimumSectionSize">
           <number>100</number>
          </attribute>
          <attribute name="horizontalHeaderDefaultSectionSize">
           <number>126</number>
          </attribute>
          <attribute name="verticalHeaderVisible">
           <bool>false</bool>
          </attribute>
          <attribute name="verticalHeaderDefaultSectionSize">
           <number>140</number>
          </attribute>
          <row/>
          <row/>
          <row/>
          <row/>
          <column/>
          <column/>
          <column/>
          <column/>
          <column/>
         </widget>
        </widget>
        <widget class="QWidget" name="menu6">
         <widget class="QTableWidget" name="tableWidget_6">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>760</width>
            <height>450</height>
           </rect>
          </property>
          <property name="focusPolicy">
           <enum>Qt::NoFocus</enum>
          </property>
          <property name="editTriggers">
           <set>QAbstractItemView::NoEditTriggers</set>
          </property>
          <property name="selectionMode">
           <enum>QAbstractItemView::SingleSelection</enum>
          </property>
          <property name="rowCount">
           <number>4</number>
          </property>
          <property name="columnCount">
           <number>5</number>
          </property>
          <attribute name="horizontalHeaderVisible">
           <bool>false</bool>
          </attribute>
          <attribute name="horizontalHeaderMinimumSectionSize">
           <number>100</number>
          </attribute>
          <attribute name="horizontalHeaderDefaultSectionSize">
           <number>126</number>
          </attribute>
          <attribute name="verticalHeaderVisible">
           <bool>false</bool>
          </attribute>
          <attribute name="verticalHeaderDefaultSectionSize">
           <number>140</number>
          </attribute>
          <row/>
          <row/>
          <row/>
          <row/>
          <column/>
          <column/>
          <column/>
          <column/>
          <column/>
         </widget>
        </widget>
       </widget>
       <widget class="QWidget" name="widget" native="true">
        <property name="geometry">
         <rect>
          <x>70</x>
          <y>0</y>
          <width>650</width>
          <height>80</height>
         </rect>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <widget class="QWidget" name="horizontalLayoutWidget">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>651</width>
           <height>81</height>
          </rect>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QPushButton" name="menuButton_1">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>60</height>
             </size>
            </property>
            <property name="focusPolicy">
             <enum>Qt::NoFocus</enum>
            </property>
            <property name="text">
             <string>한식</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="menuButton_2">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>60</height>
             </size>
            </property>
            <property name="focusPolicy">
             <enum>Qt::NoFocus</enum>
            </property>
            <property name="text">
             <string>백반</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="menuButton_3">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>60</height>
             </size>
            </property>
            <property name="focusPolicy">
             <enum>Qt::NoFocus</enum>
            </property>
            <property name="text">
             <string>분식</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="menuButton_4">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>60</height>
             </size>
            </property>
            <property name="focusPolicy">
             <enum>Qt::NoFocus</enum>
            </property>
            <property name="text">
             <string>양식</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="menuButton_5">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>60</height>
             </size>
            </property>
            <property name="focusPolicy">
             <enum>Qt::NoFocus</enum>
            </property>
            <property name="text">
             <string>일식</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="menuButton_6">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>60</height>
             </size>
            </property>
            <property name="focusPolicy">
             <enum>Qt::NoFocus</enum>
            </property>
            <property name="text">
             <string>음료</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
       <widget class="QWidget" name="payWidget" native="true">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>550</y>
          <width>800</width>
          <height>200</height>
         </rect>
        </property>
        <widget class="QListWidget" name="selectMenuList">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>0</y>
           <width>491</width>
           <height>200</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>18</pointsize>
          </font>
         </property>
         <property name="focusPolicy">
          <enum>Qt::ClickFocus</enum>
         </property>
         <property name="editTriggers">
          <set>QAbstractItemView::NoEditTriggers</set>
         </property>
        </widget>
        <widget class="QPushButton" name="payButton">
         <property name="geometry">
          <rect>
           <x>579</x>
           <y>130</y>
           <width>210</width>
           <height>71</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>26</pointsize>
          </font>
         </property>
         <property name="focusPolicy">
          <enum>Qt::NoFocus</enum>
         </property>
         <property name="text">
          <string>결제하기</string>
         </property>
        </widget>
        <widget class="QPushButton" name="deleteMenuButton">
         <property name="geometry">
          <rect>
           <x>700</x>
           <y>0</y>
           <width>90</width>
           <height>80</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>18</pointsize>
          </font>
         </property>
         <property name="focusPolicy">
          <enum>Qt::NoFocus</enum>
         </property>
         <property name="text">
          <string>삭제</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_2">
         <property name="geometry">
          <rect>
           <x>610</x>
           <y>30</y>
           <width>60</width>
           <height>16</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>16</pointsize>
          </font>
         </property>
         <property name="text">
          <string>수량</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
        <widget class="QPushButton" name="addCountButton">
         <property name="geometry">
          <rect>
           <x>600</x>
           <y>0</y>
           <width>81</width>
           <height>32</height>
          </rect>
         </property>
         <property name="focusPolicy">
          <enum>Qt::NoFocus</enum>
         </property>
         <property name="text">
          <string>추가</string>
         </property>
        </widget>
        <widget class="QPushButton" name="deleteCountButton">
         <property name="geometry">
          <rect>
           <x>600</x>
           <y>50</y>
           <width>81</width>
           <height>32</height>
          </rect>
         </property>
         <property name="focusPolicy">
          <enum>Qt::NoFocus</enum>
         </property>
         <property name="text">
          <string>빼기</string>
         </property>
        </widget>
        <widget class="QLabel" name="money_label">
         <property name="geometry">
          <rect>
           <x>610</x>
           <y>100</y>
           <width>111</width>
           <height>21</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>20</pointsize>
           <underline>true</underline>
          </font>
         </property>
         <property name="text">
          <string>0</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
        <widget class="QLabel" name="won_label">
         <property name="geometry">
          <rect>
           <x>730</x>
           <y>100</y>
           <width>21</width>
           <height>20</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>18</pointsize>
          </font>
         </property>
         <property name="text">
          <string>원</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTop|Qt::AlignTrailing</set>
         </property>
        </widget>
        <widget class="QListWidget" name="selectMenuCountList">
         <property name="geometry">
          <rect>
           <x>510</x>
           <y>0</y>
           <width>61</width>
           <height>201</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>18</pointsize>
          </font>
         </property>
        </widget>
       </widget>
      </widget>
      <widget class="QWidget" name="pay">
       <widget class="QLabel" name="label">
        <property name="geometry">
         <rect>
          <x>250</x>
          <y>300</y>
          <width>300</width>
          <height>100</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>50</pointsize>
         </font>
        </property>
        <property name="text">
         <string>주문 완료</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
    <item row="0" column="0" alignment="Qt::AlignHCenter|Qt::AlignVCenter">
     <widget class="QWidget" name="titleWidget" native="true">
      <property name="minimumSize">
       <size>
        <width>800</width>
        <height>80</height>
       </size>
      </property>
      <property name="baseSize">
       <size>
        <width>800</width>
        <height>80</height>
       </size>
      </property>
      <widget class="QLabel" name="title">
       <property name="geometry">
        <rect>
         <x>250</x>
         <y>20</y>
         <width>320</width>
         <height>40</height>
        </rect>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:24pt;&quot;&gt;학식당&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
      <widget class="QPushButton" name="homeButton">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>10</y>
         <width>60</width>
         <height>60</height>
        </rect>
       </property>
       <property name="text">
        <string>home</string>
       </property>
      </widget>
     </widget>
    </item>
    <item row="2" column="0" alignment="Qt::AlignHCenter|Qt::AlignVCenter">
     <widget class="QWidget" name="bottomWidget" native="true">
      <property name="minimumSize">
       <size>
        <width>800</width>
        <height>40</height>
       </size>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="QLabel" name="label_3">
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>40</height>
          </size>
         </property>
         <property name="text">
          <string>BY Boogie Voice</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
